theme/static/style/style.css

```diff
@@ -371,6 +375,120 @@ pre code {
     font-size: 3.5rem;
 }
 
+/* Newsletter Form Section */
+
+#mc_embed_signup form {
+    text-align: center;
+    padding: 0.10rem 0 0.10rem 0;
+}
+
+.mc-field-group {
+    display: inline-block;
+}
+
+#mc_embed_signup input.email {
+    font-family: "Open Sans", "Helvetica Neue", Arial, Helvetica, Verdana, sans-serif;
+    border: 0.1rem solid #ABB0B2;
+    -webkit-border-radius: 0.3rem;
+    -moz-border-radius: 0.3rem;
+    box-sizing: border-box;
+    padding: 0 0.4em;
+    display: inline-block;
+    width: 35rem;
+    vertical-align: top;
+    border-radius: 0;
+    margin: 0;
+    height: 5.2rem;
+    font-size: 2rem;
+}
+
+#mc_embed_signup label {
```

Comment:
```
#mc_embed_signup label {
    display: block;
    font-weight: bold;
    font-size: 3rem;
    margin-bottom: 2rem;
}
```
and move this definition above the one for the input field to follow the order in which items appear on the screen.